VideoWeb/VideoWeb/ClientApp/src/app/security/auth-base.guard.spec.ts

```diff
@@ -47,40 +42,11 @@ describe('authguard', () => {
             // Act
             let result = false;
             authGuard.canActivate(null, null).subscribe(canActivate => (result = canActivate));
-            launchDarklyServiceSpy.getFlag.withArgs(FEATURE_FLAGS.multiIdpSelection).and.returnValue(of(false));
             isAuthenticatedSubject.next(true);
             tick();
 
             // Assert
             expect(result).toBeTruthy();
         }));
     });
-
-    describe('when login failed with unsuccessful authentication', () => {
```

Comment:
the "flag: false" test case for this test is no longer relevant, but the "flag: true" path is. Please could we reinstate it